config/samples/elasticsearch/elasticsearch.yaml

```
@@ -30,6 +29,9 @@ spec:
         #   securityContext:
         #     privileged: true
         #   command: ['sh', '-c', 'sysctl -w vm.max_map_count=262144']
+        ###
+        # uncomment the line below if using a service-mesh, such as linkerd2
```

AFAIK, Istio doesn't require this so I think the wording should be changed to avoid giving the impression that all service mesh implementations need this field to be set.

```suggestion
        # uncomment the line below if you are using a service mesh such as linkerd2 that uses service account tokens for pod identification.
```

---

Your text is better